tfhe/src/integer/gpu/server_key/radix/bitwise_op.rs

```diff
@@ -9,6 +10,7 @@ use crate::integer::gpu::server_key::CudaBootstrappingKey;
 use crate::integer::gpu::{
     unchecked_bitop_integer_radix_kb_assign_async, BitOpType, CudaServerKey, PBSType,
 };
+use crate::shortint::PaddingBit;
```

Comment:
what's the current thinking on the PaddingBit thing ?

Reply:
Hum, the main reeason it exists is because some things allow to encrypt without padding bit, even in 99% of the code/ usage only work with a padding bit

Reply:
yep to keep